protocol/xmrmaker/net.go

```diff
@@ -53,11 +54,16 @@ func (b *Instance) initiate(
 		delete(b.swapStates, offer.GetID())
 	}()
 
+	symbol, err := pcommon.AssetSymbol(b.backend, offer.EthAsset)
+	if err != nil {
+		return nil, err
+	}
+
 	log.Info(color.New(color.Bold).Sprintf("**initiated swap with ID=%s**", s.ID()))
 	log.Info(color.New(color.Bold).Sprint("DO NOT EXIT THIS PROCESS OR FUNDS MAY BE LOST!"))
 	log.Infof(color.New(color.Bold).Sprintf("receiving %v %s for %v XMR",
```

Comment:
ANSI escape codes can be nice when logs are sent to a color console, but problematic in log files, especially when parsing them. Does the user have a way to disable the use of ANSI escape sequences?

Reply:
I don't think so at this point. I can look into the `color` library and see